src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationMiddleware.cs

```diff
@@ -33,23 +33,7 @@ public class GoogleAuthenticationMiddleware : OAuthAuthenticationMiddleware<Goog
             [NotNull] IOptions<ExternalAuthenticationOptions> externalOptions,
             [NotNull] IOptions<GoogleAuthenticationOptions> options,
             ConfigureOptions<GoogleAuthenticationOptions> configureOptions = null)
-            : base(next, dataProtectionProvider, loggerFactory, encoder, externalOptions, options, configureOptions)
-        {
-            if (Options.Notifications == null)
-            {
-                Options.Notifications = new GoogleAuthenticationNotifications();
-            }
-
-            if (Options.Scope.Count == 0)
```

Comment:
@Tratcher it was your idea, actually: https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationMiddleware.cs#L47

I think it's better to have it directly in the options, so the user can add additional scopes without making the base scopes "automagically" disappear. If you don't want to use the basic scope, you can clear `Options.Scope`.